code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -441,6 +441,25 @@
 	do_sparks(3, 1, src)
 	..()
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/floorbot/disassemble()
+	on = FALSE
+	var/turf/Tsec = get_turf(src)
+	var/obj/item/storage/toolbox/mechanical/N = new /obj/item/storage/toolbox/mechanical(Tsec)
```

Comment:
This one too please!

Reply:
I assume it is for the `Tsec` too ?

Reply:
`var/obj/item/storage/toolbox/mechanical/N`, single-letter variables are something we're trying to phase out!

Reply:
Okay!

Reply:
Renamed the variable to `empty_toolbox` and Tsec to `disassemble_turf`

Reply:
The `N` variable to `empty_toolbox` *